### PR TITLE
summarise profile changes in MELS

### PR DESCRIPTION
**Pull request by ara4n**
fixes https://github.com/vector-im/riot-web/issues/3463

### CODE REVIEW

**Comment by ara4n** [comment]
@lukebarnard1 the MELS code is a thing of beauty and a joy to hack on :D

File: src/components/views/elements/MemberEventListSummary.js [hunk 1]
```diff
@@ -289,7 +291,24 @@ module.exports = React.createClass({
...
+                        e.mxEvent.getPrevContent().displayname)
+                    {
+                        return 'changed_name';
+                    }
+                    else if (e.mxEvent.getContent().avatar_url !==
+                        e.mxEvent.getPrevContent().avatar_url)
+                    {
+                        return 'changed_avatar';
+                    }
+                    console.info("MELS ignoring duplicate membership join event");
```
Comment (lukebarnard1) [inline src/components/views/elements/MemberEventListSummary.js]: This looks spammy but if we really need this, I'd prefer `console.log`

**Comment by lukebarnard1** [comment]
@ara4n thanks, me and @richvdh put a lot of work into it :smile: 

LGTM apart from one little thing

**Comment by ara4n** [comment]
sorted; thanks.

**Pull request merged**